templates/x86/test/buildgen/build.conf

```diff
@@ -0,0 +1,5 @@
+TARGET = embox
+ARCH = x86
```

Comment:
I like your idea to preserve mk/.cache for CI.

Could it be like `ARCH = foo`, `ARCH = buildgen` or just empty? I'd propose also to move this config to `templates/buildgen` because it does not depend on x86.

Reply:
It's there because only `x86` have `test` subfolder :)
I would keep global template namespace as simple as possible. At least it should be `test/buildgen'

Reply:
I am okay with test/buildgen then since `x86/test/buildgen` which then used to build let's say mips/qemu look a bit weird :)

Reply:
Or you do not agree?

Reply:
> I would keep global template namespace as simple as possible

Or you can just keep this `buildgen` to be local for scripts/continuous, and then just copy it to conf/ right before `make buildgen`.